src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AmendVideoBookingService.kt

```diff
@@ -41,6 +42,11 @@ class AmendVideoBookingService(
           "The booking type ${it.bookingType} does not match the requested type ${request.bookingType}."
         }
       }
+      .also {
+        require(it.statusCode != StatusCode.CANCELLED) {
```

Comment:
There is some helper function, meh.

`requireNot(it.isStatus(StatusCode.CANCELLED)) { }`